samples/spring-security-xsuaa-usage/pom.xml

```diff
@@ -48,12 +48,26 @@
 
 	<build>
 		<finalName>${project.artifactId}</finalName>
-
 		<plugins>
 			<plugin>
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 			</plugin>
+			<plugin>
+				<groupId>org.owasp</groupId> <!--scans for vulnerabilities-->
+				<artifactId>dependency-check-maven</artifactId>
+				<version>5.2.2</version>
```

Comment:
Is a version required here?

Reply:
Since the parent of this sample is the spring-boot-starter-parent it does not inherit any version information for the dependency-check-maven plugin. So if you don't specify a version here maven will use the latest version available. If this is okay we can do that.